core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -849,6 +917,11 @@ public static void toXContent(IndexMetaData indexMetaData, XContentBuilder build
             }
             builder.endObject();
 
+            builder.startArray("primary_terms");
```

Comment:
that's how that method used to be (many more values here) - i'll change them all...